exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/types/BooleanColumnTypeTests.kt

```diff
@@ -49,4 +49,60 @@ class BooleanColumnTypeTests : DatabaseTestsBase() {
             assertEquals(idTrue, resultTrue?.get(BooleanTable.id))
         }
     }
+
+    @Test
+    fun testCustomCharBooleanColumnType() {
+        val tester = object : Table("tester") {
+            val charBooleanColumn = charBoolean("charBooleanColumn")
+            val charBooleanColumnWithDefault = charBoolean("charBooleanColumnWithDefault")
+                .default(false)
+        }
+
+        withDb {
+            try {
+                SchemaUtils.create(tester)
+
+                tester.insert {
+                    it[charBooleanColumn] = true
+                }
+
+                assertEquals(
+                    1,
+                    tester.select(tester.charBooleanColumn)
+                        .where { tester.charBooleanColumn eq true }
+                        .andWhere { tester.charBooleanColumnWithDefault eq false }
+                        .count()
+                )
+            } finally {
+                SchemaUtils.drop(tester)
+            }
+        }
+    }
 }
+
+class CharBooleanColumnType(
```

Comment:
Would it better to define such a class inside the test class, in other case it would be mixed with other global column types?

Reply:
The `exposed-tests` module is [excluded](https://github.com/JetBrains/Exposed/blob/8bac6a03deb60155fda0558374258e07490ddb26/build.gradle.kts#L18) when we publish packages during the release.

Reply:
yep, but it could be visible inside the whole test module also. In this particular case it could be suggested by IDE on every attempt to add a `bool` column.

Reply:
Ah, good point. Thanks 👍🏼